overlord/servicestate/servicestate.go

```diff
@@ -42,14 +45,93 @@ import (
 	"github.com/snapcore/snapd/wrappers"
 )
 
+type UserScope struct {
```

Comment:
this probably merits a doc comment,  also maybe UserSelector?

Reply:
Super nice suggestion, updated this!